server/public/styles.css

```diff
@@ -201,3 +202,20 @@ input[type="file"] {
 .list-padding {
   padding: 0.5rem;
 }
+
+img {
+  width: 100%;
+  height: inherit;
+}
+
+.vehicle-photo {
+  object-fit: scale-down;
+}
+
+.photo-form {
+  position: absolute;
+  top: 25%;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
```

Comment:
Rather than use absolute positioning, try margin-top or padding-top somewhere. positioning is strictly for when you want to influence the document flow.